client/packages/common/src/intl/locales/en/common.json

```diff
@@ -411,14 +414,14 @@
   "link.copy-to-clipboard": "Copy to Clipboard",
   "log.changed-from": "From",
   "log.changed-to": "to",
-  "log.invoice-created": "Shipment created",
-  "log.invoice-deleted": "Shipment deleted",
-  "log.invoice-number-allocated": "Shipment number allocated",
-  "log.invoice-status-allocated": "Shipment allocated",
-  "log.invoice-status-delivered": "Shipment delivered",
-  "log.invoice-status-picked": "Shipment picked",
-  "log.invoice-status-shipped": "Shipment shipped",
-  "log.invoice-status-verified": "Shipment verified",
+  "log.invoice-created": "Created",
+  "log.invoice-deleted": "Deleted",
+  "log.invoice-number-allocated": "Invoice number allocated",
+  "log.invoice-status-allocated": "Allocated",
```

Comment:
`Status changed to Allocated` reads gentler and is more accurate.

Reply:
[a5df294](https://github.com/msupply-foundation/open-msupply/pull/3943/commits/a5df294653813fa96746bef2fb19e891e693d0c0)